shared/templates/template_BASH_regex_permissions

```diff
@@ -0,0 +1,6 @@
+# platform = multi_platform_all, Red Hat Enterprise Linux 5
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = low
+find FILEPATH -regex '^FILEPATH/FILENAME' -exec chmod FILEMODE {} \;
```

Comment:
Why not just extend the existing `template_BASH_permissions`? And why is `chmod 0600 /boot/System.map.*` not sufficient? I don't think filepaths are really going to need crazy regex that chmod cannot handle.